src/hooks/api/places.ts

```diff
@@ -135,7 +134,6 @@ const getPlacesByQuery = async ({
   );
   const queryArguments = [
     termsString,
-    zip ? `address.\\*.postalCode:"${zip}"` : '',
```

Comment:
I think you should only omit the postalCode if the country is not belgium.

> However, for the Netherlands and Germany (not Belgium!) the postal code should be omitted in the dropdown.

Reply:
That was for the dropdown labels but I was wondering indeed it this wasn't just bringing us back to square one 🤔 Will add a condition around it